bedrock/foundation/templates/foundation/annualreport/2020/index.html

```diff
@@ -0,0 +1,729 @@
+            <a target="_blank" rel="external noopener" class="mzp-c-button mzp-t-secondary" href="https://blog.mozilla.org/en/mozilla/news/firefox-by-default-dns-over-https-rollout-in-canada/">Learn more</a>
+          </div>
+        </div>
+    </article>
+
+    <!-- Apple IDFA -->
+    <article data-modal-parent="apple-idfa" id="apple-idfa" class="mzp-c-card-feature mzp-has-aspect-16-9 mzp-l-card-feature-left-half">
+     <div class="mzp-c-card-feature-media-wrapper">
+          <div class="mzp-c-card-feature-media">
+            {{ lazy_img(image_url='img/foundation/annualreport/2020/apple-idfa-campaign.png', placeholder_url='img/foundation/annualreport/2019/spacer.gif', optional_attributes={'alt': '', 'width': '930', 'height': '523', 'class': 'mzp-c-card-image'}) }}
+          </div>
+        </div>
+        <div class="mzp-c-card-feature-content">
+          <div class="mzp-c-card-feature-content-container">
+            <img src="{{ static('img/foundation/annualreport/2019/modal-dots-2.svg') }}" alt="">
+            <h2 class="mzp-c-card-feature-title"><span class="highlight green">Apple IDFA campaign </span></h2>
+            <p class="mzp-c-card-feature-desc">Mozilla had been urging Apple to reform its app tracking policies and live up to its reputation as a privacy advocate. Apple listened, and <a href="https://foundation.mozilla.org/en/blog/applause-for-apples-idfa-decision/" target="_blank" rel="external noopener">announced the reform</a> in 2020. In 2021, that change went into effect, giving millions of consumers more privacy online.</p>
+            <a target="_blank" rel="external noopener" class="mzp-c-button mzp-t-secondary" href="https://foundation.mozilla.org/en/blog/thank-you-apple-for-standing-strong-for-privacy/">Learn more</a>
+          </div>
+        </div>
+    </article>
+
+    <!-- Venmo privacy -->
+     <article data-modal-parent="venmo-privacy" id="venmo-privacy" class="mzp-c-card-feature mzp-has-aspect-16-9 mzp-l-card-feature-left-half">
+     <div class="mzp-c-card-feature-media-wrapper">
+          <div class="mzp-c-card-feature-media">
+            {{ lazy_img(image_url='img/foundation/annualreport/2020/venmo-privacy.png', placeholder_url='img/foundation/annualreport/2019/spacer.gif', optional_attributes={'alt': '', 'width': '930', 'height': '523', 'class': 'mzp-c-card-image'}) }}
+          </div>
+        </div>
+        <div class="mzp-c-card-feature-content">
+          <div class="mzp-c-card-feature-content-container">
+            <img src="{{ static('img/foundation/annualreport/2019/modal-dots-2.svg') }}" alt="">
+            <h2 class="mzp-c-card-feature-title"><span class="highlight green">Venmo privacy campaign</span></h2>
+            <p class="mzp-c-card-feature-desc">For years, Mozilla has urged the payment app Venmo to remove its public feed, a feature that let users see transactions between total strangers. In July, Venmo finally listened — a big win for consumer privacy.</p>
```

Comment:
There's supposed to be a hyperlink around the words `Venmo finally listened` but it's the exact same link as the one on the button in this card, so I removed it. Open for the reviewer's thoughts.